src/si.c

```diff
@@ -4,7 +4,7 @@ s32 si_find_label(ScriptInstance* script, s32 arg1);
 s32 si_skip_if(ScriptInstance* script);
 s32 si_skip_else(ScriptInstance* script);
 s32 si_goto_end_loop(ScriptInstance* script);
-s32 si_goto_end_case(ScriptInstance* script);
+Bytecode* si_goto_end_case(ScriptInstance* script);
```

Comment:
:+1:

The other `si_goto` functions (should really be `si_find` IMO) will return `Bytecode*` too I expect.